src/App.vue

```diff
@@ -1,13 +1,19 @@
 <template>
   <div id="app">
-    <div id="nav">
-      <router-link to="/">Home</router-link> |
-      <router-link to="/about">About</router-link>
-    </div>
+    <TheHeader />
+    <router-link to="/" />
     <router-view/>
```

Comment:
I'm excited to learn more about how Vue uses router. It seems simpler than React.

Reply:
I agree! It looks very similar to how Redux was set up with router in our last project.